### ISSUE TITLE
Feature : Split bill option while payment

### ISSUE THREAD

**Issue by Pratik-11** [m0hit-kumar/Bring-The-Menu#52]
**Feature :**

In many cases when we go out dining with a group splitting the bill is a problem,
We can solve that by adding a split bill feature while paying the bill. This can be an important and very useful feature for the app.

**Approach :**
The approach is something like this
- At the time of payment after totaling of bill there can be a button for split in side of pay
- when clicking that a page opens which lets you decide no of people to split between 
- It can have two options like split evenly, manual 
- then a QR generates in which for example 5000 rs in between 5 ppl evenly so, at first 1000 is deducted from parent app account the QR will accept 4 payments of 1000 each and then the QR is not usefull.
- the screen would show payments received by number of people 
- at the end when all payments are recieved the parent app has the option to pay

**Restrictions :**
- There are lots of restrictions and problematic situations that can arise so i suggest having a disccussion on this feature
- we can start first with the UI then will work on it

**Comment by ViAsmit** [comment]
Great Idea! We can work on it. Please go through the app once. And If you are already preparing some HLD or docs. Do share it once. 
We can discuss it further.

**Comment by Pratik-11** [comment]
yup thanks i am currently going through the app i have got some bugs too will list them as issues too and will discuss the features before implementing
